frontend/src/booking/edit-booking-modal.vue

```diff
@@ -57,7 +57,7 @@
           <b-form-row v-if="!examAssociated">
             <b-col>
               <b-form-group>
-                <label :style="{color: labelColor}">Event Title</label><br>
+                <label :style="{color: labelColor}">Scheduling Party</label><br>
```

Comment:
Should this stay as "Event Title" when the it's an exam being booked?

Reply:
I thought that Becky said that this was a blanket change?

Reply:
https://trello.com/c/2JvqBVuD/293-editing-the-non-exam-booking-also-update-the-event-title-field-to-be-scheduling-party

Reply:
It says for "non-exam booking", but I can see how that can be ambiguous. Can you follow up with Becky to confirm?

Reply:
Yes.